src/data/grammar.json

```diff
@@ -13,10 +13,6 @@
         "#announcement#",
         "#clickbait#",
         "#flavorArcana#",
-        "#inGraphics#",
```

Comment:
I'd like to keep this cheat in until we have more types of graphical tweets, cause these are REALLY good and high quality compared to most other tweets